### ISSUE TITLE
Create L1 Data View

### ISSUE THREAD

**Issue by llamzonc** [hcastro1989/Cognitive-Delivery#14]
- L1 Number
- Month
- Cost/Rev Category
- Value
- Number of project
- Client
- Industry

**Comment by llamzonc** [comment]
Team also needs L1 earliest start date and L2 latest end date

**Comment by llamzonc** [comment]
Is the Client Name in the data? Didn't see it in the zip posted on box: https://ibm.ent.box.com/file/122368853780

**Comment by areyes13** [comment]
added customer fields

**Comment by llamzonc** [comment]
add Time and Materials vs Fixed Cost

**Comment by areyes13** [comment]
I scanned the plan / actual files for 'fixed', 'time', 'materials', etc. but I can't find an obvious field that includes this distinction. The only thing that shows up is in the Monthly Actuals file (the ones broken out by sector) is the 'Category' for rev/cost. There is a Category for **Subcontractor PO Fixed** but I'm not sure that is correct (?).

If someone else finds that field just post it in here (and which file it came from) so I can add it to the cube. Otherwise someone please email Bill to ask for which specific field includes this distinction.

**Comment by llamzonc** [comment]
@hcastro1989 did you say you had found this field? We can move this to closed once we have it incorporated

**Comment by hcastro1989** [comment]
Yes, Al said that it is reported in the Service Type column. Here are the common service types (projects) you'll find:   
FP: Fixed Priced   
BE: Best Efforts (aka T&M)
IC: Incident billings - which deal with a specific algorithm to bill the client that escapes FP or BE methods
